core/src/main/scala/fs2/Process1.scala

```diff
@@ -188,10 +188,18 @@ object process1 {
   def sum[F[_],I](implicit ev: Numeric[I]): Stream[F,I] => Stream[F,I] =
     fold(ev.zero)(ev.plus)
 
+  /** Emits all elements of the input except the first one. */
+  def tail[F[_],I]: Stream[F,I] => Stream[F,I] =
+    drop(1)
+
   /** Emit the first `n` elements of the input `Handle` and return the new `Handle`. */
   def take[F[_],I](n: Long): Stream[F,I] => Stream[F,I] =
     _ pull Pull.take(n)
 
+  /** Emits the last `n` elements of the input. */
+  def takeRight[F[_],I](n: Long): Stream[F,I] => Stream[F,I] =
```

Comment:
with the change to `Pull.takeRight`, this would become `_ pull Pull.takeRight(n).flatMap(is => Pull.output(Chunk.indexedSeq(is)))`